composer.json

```diff
@@ -35,5 +35,9 @@
         "branch-alias": {
             "dev-master": "1.0.x-dev"
         }
-    }
+    },
+    "bin": [
+        "bin/doctrine-migrations",
+        "bin/doctrine-migrations.php"
```

Comment:
`` bin/doctrine-migrations.php`` should not be there. Only the actual bin is needed